radicle-cli/src/commands/remote/add.rs

```diff
@@ -0,0 +1,41 @@
+use radicle::node::tracking::store::Config;
+use radicle::{git::Url, node::TRACKING_DB_FILE, prelude::Id, Profile};
+use radicle_crypto::PublicKey;
+
+use crate::git::rad_add_remote;
+use crate::{git, terminal as term};
+
+pub fn run(
+    repository: &git::Repository,
+    profile: &Profile,
+    pubkey: &PublicKey,
+    name: Option<String>,
+    id: Id,
+) -> anyhow::Result<()> {
+    // looks like that the clippy suggestion is wrong, and
+    // because name is not a function!
+    #[allow(clippy::redundant_closure)]
+    let Some(name) = name.map_or(get_remote_name(profile, pubkey)?, |name| Some(name)) else {
+        anyhow::bail!("an alias need to be specified");
```

Comment:
Also, this pattern is pretty common so it's possible to write:
```suggestion
    let name = name
        .map_or(get_remote_name(profile, pubkey)?, Some)
        .ok_or(anyhow::anyhow!("an alias need to be specified"))?;
```

Reply:
Thanks! sorry if I use the wrong patter it takes me a while to adapt to the repository code style!

Thanks for your review!

Reply:
My pleasure :blush: I'm here to pass on the knowledge I've gathered writing Rust myself